README.md

```diff
@@ -71,16 +70,61 @@ FIRESTORE_EMULATOR_HOST=[::1]:8195`.
 
 You can hit the endpoints with curl or Postman.
 
+Try:
+
+```
+curl --request GET 'http://localhost:8080/challenge' \
+     --header 'X-Forwarded-Proto: https'
+```
+
+### HTTPS
+
+The local dev server uses HTTP so you must send a fake HTTPS header to prevent the
+request from being rejected.
+
+If you see an error message like this with HTTP Code 426:
+
+```
+{"Error":"Please use HTTPS"}
+```
+
+You should add either:
+
+```
+X-Forwarded-Proto: https
+```
+
+or
+
+```
+Forwarded: for=\"localhost\";proto=https
+```
+
 ## Firebase Security
 
-Unauthenticated Firestore access is disabled. If you want to access the emulator
-you can send the following header to override the security with the bearer token
-"owner".
+Unauthenticated Firestore access is disabled. If you want to bypass the firestore.rules
+on the emulator REST interface just supply the auth bearer header value "owner":
 
-```text
-curl --location \
-  --request GET 'http://localhost:8080/v1/projects/covidwatch-354ce/databases/(default)/documents/challenges' \
-  --header 'Authorization: Bearer owner'
+```
+Authorization: Bearer owner
+```
+
+## Deployment
+
+You can deploy the functions with:
+
+```
+$ source ./deploy.sh
+```
+
+Or manually deploy a Go cloud function from the functions folder like so:
```

Comment:
I would remove this; they can just look at the source of `deploy.sh` if they want to know how to do it manually.